src/emulator/firestoreEmulator.ts

```diff
@@ -54,7 +55,35 @@ export class FirestoreEmulator implements EmulatorInstance {
       });
     }
 
-    return javaEmulators.start(Emulators.FIRESTORE, this.args);
+    // Make a copy of the args
+    const commandLineArgs: any = Object.assign({}, this.args);
```

Comment:
Wont have to do this stuff once webchannel_port is implemented and then we can make things less stringly typed.